examples/prom/mixed_nonlinear_diffusion.cpp

```diff
@@ -533,6 +534,8 @@ int main(int argc, char *argv[])
                    "Enable or disable the merge phase.");
     args.AddOption(&use_sopt, "-sopt", "--sopt", "-no-sopt", "--no-sopt",
```

Comment:
This is not a change in this PR, but I noticed that `--sopt` is in the sample runs, which could be `-sopt`.